### PR TITLE
Add Service Activity Model & Update Hatoba Model

### PR DESCRIPTION
**Pull request by fujita-hiroki1115**
## Summary

- Support model json for service activity
- Update hatoba model

## Test

- [x] Check test.

```
make test
```

- [x]  Run example go file.

```
package main

import (
	"context"
	"fmt"

	"github.com/nifcloud/nifcloud-sdk-go/nifcloud"
	"github.com/nifcloud/nifcloud-sdk-go/service/dns"
)

func main() {
	// Create config with credentials and region.
	cfg := nifcloud.NewConfig(
                "YOUR_ACCESS_KEY_ID",
                "YOUR_SECRET_ACCESS_KEY",
		"jp-east-1",
	)

          svc := serviceactivity.NewFromConfig(cfg)
  
          resp, err := svc.DescribeServiceStatuses(context.TODO(), nil)
          if err != nil {
                  panic(err)
          }

          fmt.Println("ServiceMenu:")
          for _, serviceMenu := range resp.Data.ServiceMenu {
                  fmt.Println(nifcloud.ToString(serviceMenu.Name))
          }
  }
```

### CODE REVIEW

File: codegen/smithy-nifcloud-go-codegen/src/main/resources/software/nifcloud/smithy/nifcloud/go/codegen/endpoints.json [hunk 1]
```diff
@@ -13,6 +13,7 @@
         ]
       },
       "dnsSuffix": "api.nifcloud.com",
+      "serviceActivitySuffix": "api.nifcloud.com",
```
Comment (aokumasan) [inline codegen/smithy-nifcloud-go-codegen/src/main/resources/software/nifcloud/smithy/nifcloud/go/codegen/endpoints.json:16]: This is invalid field. This field is never used in endpoint generator.
https://github.com/nifcloud/nifcloud-sdk-go/blob/main/codegen/smithy-nifcloud-go-codegen/src/main/java/software/nifcloud/smithy/nifcloud/go/codegen/EndpointGenerator.java

Reference: AWS botocore's endpoints.json  
https://github.com/boto/botocore/blob/develop/botocore/data/endpoints.json

Comment (fujita-hiroki1115) [reply codegen/smithy-nifcloud-go-codegen/src/main/resources/software/nifcloud/smithy/nifcloud/go/codegen/endpoints.json]: Removed this field

Comment (aokumasan) [reply codegen/smithy-nifcloud-go-codegen/src/main/resources/software/nifcloud/smithy/nifcloud/go/codegen/endpoints.json]: Thanks!

**Review by aokumasan — APPROVED** [review]

**Comment by aokumasan** [comment]
LGTM 🐱

**Pull request merged**